src/save/encodeDataURL.ts

```diff
@@ -0,0 +1,25 @@
+import { encode as uint8encode } from 'uint8-base64';
+
+import type { Image } from '../Image.js';
+
+import type {
+  EncodeOptionsBmp,
+  EncodeOptionsJpeg,
+  EncodeOptionsPng,
+} from './encode.js';
+import { encode, defaultPng } from './encode.js';
+/**
+ * Converts image into Data URL string.
+ * @param image - Image to get base64 encoding from.
+ * @param options - Encoding options.
+ * @returns base64 string.
+ */
+export function encodeDataURL(
+  image: Image,
+  options: EncodeOptionsBmp | EncodeOptionsPng | EncodeOptionsJpeg = defaultPng,
```

Comment:
I kept the types the same way they were in `encode` function but should it be a single type that combines all encode<Type>Options interfaces?